### PR TITLE
Add Orlando Chain testnet

### PR DESCRIPTION
**Pull request by orlchain**
Please.

### CODE REVIEW

File: _data/chains/eip155-3031.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "decimals": 18
+  },
+  "infoURL": "http://orlchain.com/",
+  "shortName": "ORL",
+  "chainId": 3031,
+  "networkId": 3031,
+  "icon": "orl",
+  "explorers": [{
+    "name": "Orlando (ORL) Blockchain Explorer",
+    "url": "https://orlscan.com/",
```
Comment (ligi) [inline _data/chains/eip155-3031.json:20]:
```suggestion
    "url": "https://orlscan.com",
```

File: _data/chains/eip155-3031.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "rpc": [
+    "http://rpc-testnet.orlchain.com/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Orlando",
+    "symbol": "ORL",
+    "decimals": 18
+  },
+  "infoURL": "http://orlchain.com/",
```
Comment (ligi) [inline _data/chains/eip155-3031.json:13]:
```suggestion
  "infoURL": "http://orlchain.com",
```

File: _data/chains/eip155-3031.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Orlando Chain",
+  "chain": "ORL",
+  "rpc": [
+    "http://rpc-testnet.orlchain.com/"
```
Comment (ligi) [inline _data/chains/eip155-3031.json:5]:
```suggestion
    "http://rpc-testnet.orlchain.com"
```

File: _data/icons/orl.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmNsuuBBTHErnuFDcdyzaY8CKoVJtobsLJx2WQjaPjcp7g",
```
Comment (ligi) [inline _data/icons/orl.json:3]: where did you pin this CID? I cannot resolve it

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/orl.json (hunk 4 above)
Comment (orlchain) [reply _data/icons/orl.json:3]: I made a pin on infura: https://orlando.infura-ipfs.io/ipfs/QmNsuuBBTHErnuFDcdyzaY8CKoVJtobsLJx2WQjaPjcp7g

**Review by ligi — APPROVED** [review]

**Comment by Apdlrcjafg19** [comment]
> Please.

Avances

File: _data/icons/orl.json (hunk 4 above)
Comment (orlchain) [reply _data/icons/orl.json:3]: Please check it.

**Comment by Moneymab223** [comment]
Wat to do

**Pull request merged**